Makefile

```diff
@@ -136,7 +138,8 @@ validate-docs: docs/checks/internal/generate/main.go
 	# Validating checks.yaml
 	go run ./docs/checks/internal/validate/main.go
 
-build-scorecard: ## Runs go build on repo
+build-scorecard: scorecard-ko
```

Comment:
@imjasonh Any clue as to what am I doing wrong here? I am trying to publish scorecard into ghcr.io/naveensrinivasan. Thanks

Reply:
Thanks @imjasonh 

When I did what you suggested I ran into this. https://github.com/naveensrinivasan/scorecard/runs/6029451063?check_suite_focus=true
```
O_CACHE=/tmp/ko ko build -B \
[60](https://github.com/naveensrinivasan/scorecard/runs/6029451063?check_suite_focus=true#step:9:60)
		   --sbom=none \
[61](https://github.com/naveensrinivasan/scorecard/runs/6029451063?check_suite_focus=true#step:9:61)
		   --platform="linux/amd64,linux/arm64,linux/386,linux/arm"\
[62](https://github.com/naveensrinivasan/scorecard/runs/6029451063?check_suite_focus=true#step:9:62)
		   --tags latest,v1.1.1-1107-g1b57d81,1b57d8164bd1cd20fafb40f80848[63](https://github.com/naveensrinivasan/scorecard/runs/6029451063?check_suite_focus=true#step:9:63)0c1d189d49 github.com/ossf/scorecard/v4
63
2022/04/14 19:10:14 Using base gcr.io/distroless/static:nonroot@sha256:2556293984c5738fc75208cce52cf0a4762c709cf38e4bf8def65a61992da0ad for github.com/ossf/scorecard/v4
[64](https://github.com/naveensrinivasan/scorecard/runs/6029451063?check_suite_focus=true#step:9:64)
2022/04/14 19:10:14 Using build config scorecard for github.com/ossf/scorecard/v4
[65](https://github.com/naveensrinivasan/scorecard/runs/6029451063?check_suite_focus=true#step:9:65)
2022/04/14 19:10:14 Using build config scorecard for github.com/ossf/scorecard/v4
[66](https://github.com/naveensrinivasan/scorecard/runs/6029451063?check_suite_focus=true#step:9:66)
2022/04/14 19:10:14 Building github.com/ossf/scorecard/v4 for linux/amd64
[67](https://github.com/naveensrinivasan/scorecard/runs/6029451063?check_suite_focus=true#step:9:67)
2022/04/14 19:10:14 Building github.com/ossf/scorecard/v4 for linux/arm
[68](https://github.com/naveensrinivasan/scorecard/runs/6029451063?check_suite_focus=true#step:9:68)
2022/04/14 19:10:17 Using build config scorecard for github.com/ossf/scorecard/v4
[69](https://github.com/naveensrinivasan/scorecard/runs/6029451063?check_suite_focus=true#step:9:69)
2022/04/14 19:10:17 Building github.com/ossf/scorecard/v4 for linux/arm64
[70](https://github.com/naveensrinivasan/scorecard/runs/6029451063?check_suite_focus=true#step:9:70)
2022/04/14 19:10:18 Publishing ghcr.io/naveensrinivasan/scorecard/v4:latest
[71](https://github.com/naveensrinivasan/scorecard/runs/6029451063?check_suite_focus=true#step:9:71)
Error: failed to publish images: error publishing ko://github.com/ossf/scorecard/v4: POST https://ghcr.io/v2/naveensrinivasan/scorecard/v4/blobs/uploads/: DENIED: installation not allowed to Create organization package
[72](https://github.com/naveensrinivasan/scorecard/runs/6029451063?check_suite_focus=true#step:9:72)
2022/04/14 19:10:19 error during command execution:failed to publish images: error publishing ko://github.com/ossf/scorecard/v4: POST https://ghcr.io/v2/naveensrinivasan/scorecard/v4/blobs/uploads/: DENIED: installation not allowed to Create organization package```

AFAIK it is because of `v4`. How do I avoid that?

Thanks